test.rb

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env ruby
+require 'logatron/logatron'
+
+Logatron.info("Log entry from the 'testing script.'  Single quotes in the body of this message should be converted into double quotes.")
```

Comment:
Also, you have a spec test `configuration_spec.rb` that tests the output. Why do you need this file for a docker build?